### PR TITLE
Updates Ant to 1.10.14, and maven-plugin-api to 3.9.4

### PR DESCRIPTION
**Pull request by kterusaki**
- org.apache.ant:ant:1.9.9 - [CVE-2020-11979, CVE-2021-36373, CVE-2021-36374, CVE-2020-194]
- org.codehaus.plexus:plexus-utils:3.0.22 - [CVE-2022-4244, CVE-2022-4245]

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -38,13 +38,13 @@ lazy val jarjar = project
 
     IntegrationTest / fork := true
     IntegrationTest / envVars := Map(
-      "JARJAR_CLASSPATH" -> (Runtime / fullClasspath)
```
Comment (kterusaki) [inline build.sbt:41]: L41-47 are formatting changes that weren't applied previously

**Review by eed3si9n — APPROVED** [review]
Thanks!

**Comment by kterusaki** [comment]
@eed3si9n what's the process to release a new version that contains these changes? I'd like to update the jarjar abrams version used in sbt-assembly

**Comment by eed3si9n** [comment]
I'll try to get one out this week.

**Comment by eed3si9n** [comment]
FYI, I did release a new version, but I think a better approach would be to split out Ant - https://github.com/eed3si9n/jarjar-abrams/pull/36

**Pull request merged**